designer_v2/lib/localization/app_de.arb

```diff
@@ -193,11 +198,16 @@
   "form_field_study_title_required": "Die Studie muss einen Titel haben",
   "form_field_study_title_default": "Unbenannte Studie",
   "form_field_study_description": "Beschreibung",
-  "form_field_study_description_tooltip": "Beschreibe die Studie kurz & verständlich für Teilnehmer",
-  "form_field_study_description_hint": "Beschreibe die Studie kurz & verständlich für Teilnehmer",
+  "form_field_study_description_tooltip": "Beschreibe die Studie kurz und verständlich für Teilnehmer",
+  "form_field_study_description_hint": "Beschreibe die Studie kurz und verständlich für Teilnehmer",
```

Comment:
The arb files also still have other ampersands; would also consider changing all of them if they were a problem here